onnxruntime/core/optimizer/embed_layer_norm_fusion.cc

```diff
@@ -732,11 +735,7 @@ static bool FuseSubGraph(Graph& graph,
   nodes_to_remove.push_back(layer_norm_add_node.Index());
   nodes_to_remove.push_back(layer_norm_node.Index());
 
-  for (const auto& index : nodes_to_remove) {
-    Node* node = graph.GetNode(index);
-    graph_utils::RemoveNodeOutputEdges(graph, *node);
-    graph.RemoveNode(node->Index());
-  }
+  RemoveWithSharedNodeInTop(graph, nodes_to_remove);
```

Comment:
Is it possible to use graph_utils::RemoveNodesWithOneOutputBottomUp for remove nodes?